pages/common/sftp.md

```diff
@@ -0,0 +1,20 @@
+# sftp
+
+> Secure file transfer program.
+> Copy files between hosts using Secure File Transfer Protocol over SSH.
```

Comment:
Modified the first example to explicitly say "and enter an interactive command mode" which I think is better than trying to stuff "interactive" into the program description.